README.md

```diff
@@ -74,7 +74,7 @@ Then automatically add it to your jest tests by using `setupFilesAfterEnv` optio
 
 ### Custom Jest Preset
 
-> **important** if you use "modern" Fake Timers
+> **important** if you use "modern" Fake Timers and React Native < 0.71.2
```

Comment:
This commit is available in all 0.71 variants
<img width="798" alt="image" src="https://user-images.githubusercontent.com/5106466/218985643-61b9f294-0c7e-4212-8097-9fefea63f4fd.png">

Reply:
Oh I hadn't seen this, I updated the docs accordingly, thanks!